wire-compiler/src/main/java/com/squareup/wire/WireCompiler.java

```diff
@@ -176,6 +188,13 @@ static WireCompiler forArgs(
         for (String identifier : splitArg(arg, EXCLUDES_FLAG.length())) {
           identifierSetBuilder.exclude(identifier);
         }
+      } else if (arg.startsWith(CUSTOM_PROTO_ADAPTER)) {
+        List<String> customProtoAdapterArgs = splitArg(arg, CUSTOM_PROTO_ADAPTER.length());
+        String protoPath = customProtoAdapterArgs.get(0).trim();
```

Comment:
what does _path_ mean here? Is it the name? Typically _path_ is a directory, or a list of directories, but this is just a fully qualified name.